package.json

```diff
@@ -15,31 +15,22 @@
   },
   "scripts": {
     "start": "watchman watch-del-all && export DEV_MODE=true && react-native start",
-    "start-ios": "node ./scripts/cliStuff.js",
     "ios": "react-native run-ios",
-    "iPad": "react-native run-ios --simulator='iPad Pro (9.7 inch)'",
     "android": "react-native run-android",
+    "iPad": "react-native run-ios --simulator='iPad Pro (9.7 inch)'",
     "pretest": "npm run lint",
     "test": "jest",
     "test:watch": "jest --watch",
     "lint": "eslint src -c .eslintrc.js --ext .tsx,.ts,.js",
     "lint:fix": "eslint src -c .eslintrc.js --fix",
-    "lint:test": "mocha --compilers js:babel-core/register eslint-rules/tests/lib/rules",
     "xcode": "xed ios",
     "build:dev": "tsc --p tsconfig.dev.json",
-    "build:src": "./node_modules/.bin/babel src --out-dir src --config-file ./src/.babelrc.json --extensions '.ts,.tsx' --ignore 'src/index.ts'",
-    "build:lib": "./node_modules/.bin/babel lib --out-dir lib --config-file ./src/.babelrc.json --extensions '.ts,.tsx'",
-    "build:exports": "npm run build:src && npm run build:lib",
-    "build:packages": "node scripts/buildPackages.js",
```

Comment:
Isn't `build:packages` useful when adding a new package?

Reply:
It's part of `scripts/build` script

Reply:
I know